src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area, Drawable, Randomizable {
```

Comment:
The `Figure` class should not implement interfaces that have conceptually different methods such as `Area` and `Drawable`. It's better to separate these concerns into different interfaces and implement them in the specific classes that require those behaviors.